.github/actions/install-requirement/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Install Project Requirements
+description: Install requirements from a PEP 517 project
+
+inputs:
+  requirements:
+    description: "Whitespace-delimited list of project requirements to install."
```

Comment:
Given it's being specified as a list, it would be nice if this was an actual list/array, rather than needing to do the whitespace-interpretation dance. AFAICT all that should be required is a `{{ join(inputs.requirements, " " }}` where it's used?

Reply:
AFAIR, GitHub doesn't allow us to use complex parameters in to these types of Actions; all inputs are strings. (On a related note, that's also why I have to pass `"true"` to Hynek's Action for attestation instead of just `true`.)

Reply:
I think this is a limitation of the entire framework for actions regardless of the type.

For instance, consider `actions/setup-python`:
```yaml
steps:
- uses: actions/checkout@v4
- uses: actions/setup-python@v5
  with:
    python-version: '3.10'
    cache: 'pip'
    cache-dependency-path: |
      **/setup.cfg
      **/requirements*.txt
```
They use `|` for `cache-dependency-path` to make it feel like a list.....but that's just fancy YAML for space-delimited list.

So, I think array or list support is limited to YAML that the core GitHub Actions infrastructure processes.